desktop-widgets/mainwindow.cpp

```diff
@@ -1166,24 +1166,17 @@ void MainWindow::on_actionViewAll_triggered()
 {
 	toggleCollapsible(false);
 	beginChangeState(VIEWALL);
-	static QList<int> mainSizes;
+
 	const int appH = qApp->desktop()->size().height();
 	const int appW = qApp->desktop()->size().width();
-	if (mainSizes.empty()) {
-		mainSizes.append(lrint(appH * 0.7));
-		mainSizes.append(lrint(appH * 0.3));
-	}
-	static QList<int> infoProfileSizes;
-	if (infoProfileSizes.empty()) {
-		infoProfileSizes.append(lrint(appW * 0.3));
-		infoProfileSizes.append(lrint(appW * 0.7));
-	}
 
-	static QList<int> listGlobeSizes;
-	if (listGlobeSizes.empty()) {
-		listGlobeSizes.append(lrint(appW * 0.7));
-		listGlobeSizes.append(lrint(appW * 0.3));
-	}
+    auto _lrint = [](double value) {
+        return static_cast<int>(std::lrint(value));
+    };
```

Comment:
umm, what's that? How is this better than a macro?

Reply:
(a) this is a lambda function and @tcanabrava know just how much I hate this particular piece of syntax licorice (just like licorice (Lakritze) this pretends to be candy but is just flat out disgusting and vomit inducing, the same is IMHO true for this abuse of letters) 
(b) which is why I suggested using a macro (or, if you absolutely have to) a static function
(c) the difference is in rounding behavior, IIRC - we introduced lrint on purpose to fix some really odd bugs

Reply:
> the difference is in rounding behavior, IIRC - we introduced lrint on purpose to fix some really odd bugs

Yeah, `rint` returns a `double` not an `int` as I inferred from it's name.

Reply:
1 - I'll change this to a proper function
2 - I'll buy you Salmiakki, that's Licorice with amonium chloride <3